crates/wasm-pkg-core/src/config.rs

```diff
@@ -77,7 +77,7 @@ pub struct Override {
 pub struct Metadata {
     /// The author(s) of the package.
     #[serde(default, skip_serializing_if = "Option::is_none")]
```

Comment:
Can you please add an alias for `author` so the old versions of file still parse correctly?

Reply:
Good idea, done